.github/actions/md-toc/action.yml

```diff
@@ -6,7 +6,9 @@ inputs:
     required: false
     default: 1.2.0
   md_src:
-    description: Markdown file to generate Table of Contents
+    description: |
+      Markdown file(s) to generate Table of Contents for. It can be a list of
+      space separated files or a glob pattern relative to the repository root.
```

Comment:
but you have explicitly disabled globbing in git-auto-commit-action

Reply:
Looks like it works https://github.com/Alfresco/flux-alfresco-pipeline/actions/runs/12184345830/job/33988003648#step:3:124